src/constants/request.ts

```diff
@@ -69,8 +69,8 @@ export const URLs = {
   notes: {
     get: '/notes',
     create: '/notes',
-    update: '/notes',
-    delete: '/notes'
+    update: '/notes/:noteId',
+    delete: '/notes/:noteId'
```

Comment:
I think it's redundant to specify `note` on each URL.
It's already understandable whos `id` is that. 
Also, you should specify the names of IDs if there is more than 1 ID in the URL.
```suggestion
    update: '/notes/:id',
    delete: '/notes/:id'
```